pkg/token/helper/tokenHelper.go

```diff
@@ -101,7 +100,7 @@ func GenerateRefreshToken(user *model.User, secretKey string, expirationInSecond
 	currentTime := time.Now()
 	tokenExpiration := currentTime.Add(time.Duration(expirationInSeconds) * time.Second)
 
-	tokenId, err := uuid.NewV4()
+	tokenId, err := uuid.NewUUID()
```

Comment:
You can use `uuid.NewString()` https://pkg.go.dev/github.com/google/uuid#NewString as you only want a string.